anasymod/config.py

```diff
@@ -340,6 +340,11 @@ def __init__(self, cfg_file):
             sometimes has bugs, particularly when more complex Verilog features like hierarchical references and
             interfaces are used."""
 
+        self.vivado_stack = None
+        """ type(int) : Stack size used by Vivado.  If you are getting an error during synthesis that
+            looks like "Segmentation fault "$RDI_PROG"", try setting this to 2000 or higher
+            (https://www.xilinx.com/support/answers/64434.html)."""
+

```

Comment:
I found this option, in combination with a large swap space, enabled **anasymod** to build bitstreams for ZCU106 on my old computer, which has limited memory.  It's not fast, but at least it prevents Vivado from crashing.